### ISSUE TITLE
Maximo, Business Process, Pavement Operations

### ISSUE THREAD

**Issue by adry-martinez** [cityofaustin/atd-data-tech#13860]
- [x] write Pavement Ops SOP
- [ ] discuss Start Center design
- [ ] review SOP with Division Manager and Project Coordinator
- [ ] present SOP to Pavement Ops staff after approval
- [ ] upload SOP to sharepoint

**Comment by adry-martinez** [comment]
Review slides for guidance [cityofaustin.sharepoint.com/:p:/r/sites/SIM-MaximoDiscussion/Shared%20Documents/General/SBO%20Maximo/4%20-%20Pavement%20Operations/PWD-Maximo-Seal%20Coat%20and%20Crack%20Seal%20(PM%20and%20Prep)%20Work%20Orders.pptx?d=w3dda7f299ed2467ab39772a3c1233867&csf=1&web=1&e=acCwU0](url)

**Comment by adry-martinez** [comment]
10/9 
Discussed with DM and PC regarding creating separate start centers
Fog Seal and Sealcoat will be separate with Crack seal underneath.
Overlay will also be a standalone for their start center
Also consideration for DM to have their designated Start Center to manage

**Comment by adry-martinez** [comment]
requesting to add restriction to SEALCOAT_WORK work orders that requires Length and Width to be required before 4COMP, will need to include CTM in the discussion.

**Comment by adry-martinez** [comment]
request to change the name of seal coat 2 to fog seal and request to consider a restriction on work orders from being closed-out if pictures are not attached

**Comment by adry-martinez** [comment]
Find a way to collect information when a project is delayed. 
Using Specifications tab to say whether this was unproductive or productive, # of days the project was delayed using actual/scheduled time, reason for delay (WEATHER, EQUIPMENT FAILURE, UNDERSTAFFED, ETC)

**Comment by rebekkabry** [comment]
# Agenda Prep 
**💭  notes only, agenda build out in process)** 

- [Scratch Paper Miro](https://miro.com/app/board/uXjVKI0wi4c=/) - work order high level with phases 
- Pavement Work Order Completion Needs #16771
- PPT Presentation from Maximo Team (get link inserted)

**Comment by rebekkabry** [comment]
@ReneVarg25 Do you feel this is still needed?

**Comment by rebekkabry** [comment]
@ReneVarg25 no. we can update the processes documentation when we update to MAS9 if needed. Will close this out.
